about.html

```diff
@@ -0,0 +1,129 @@
+                <li><a href="">Program</a></li>
+                <li><a href="">Join</a></li>
+                <li><a href="">Sponsor</a></li>
+                <li><a href="">News</a></li>
+                <li><a class="campaign" href="">CC Campaign</a></li>
+            </ul>
+        </nav>
+    </dialog>
+    <main class="container">
+        <section class="s_headline">
+            <div class="wrap">
+                <div class="wrapper1">
+                    <h1>DEFCON HACKER CONFERENCE 2023</h1>
+                    <hr class="line">
+                    <div class="description">
+                        <p class="des_mob">DEF CON started as talks and parties that evolved into a community of those who with a hacker’s mindset and an insatiable curiosity of how things work. We are committed to creating a place for all hackers to share, interact and learn from each other.</p>
+                        <p class="des_desk">DEF CON started as talks and parties that evolved into a community of those who with a hacker’s mindset and an insatiable curiosity of how things work. We are committed to creating a place for all hackers to share, interact and learn from each other. Sometimes, though, we have to reassess whether the structure of DEF CON is helping or hindering this vision.
+
+One example of this is how we categorize creators. There are a lot of content departments at DEF CON. If you’re a Vendor, you go here. If you’re a Village, you go over there. If something doesn’t quite fit our categories we force them to fit: The EFF or a University in our vendor room for example.</p>
+                    </div>
+                    <div class="address">
+                        <p>Please contact us per Email for further questions about DEFCON Conference 2023</p>
+                        <p class="email">defcon2023@defcon.org <span>&gt;</span></p>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section class="s_logo">
+                <div class="wrapper_logo">
+                    <h2>DEFCON HACKER<br>CONFERENCE 2023 Logo</h2>
+                    <hr class="line">   
+                    <p>The logo of DEFCON hacker conference 2023 was decided through the logo competition from 8, June to 7, July.</p>
+                    <img src="static/images/defcon_logo.png" alt="CC global summit logo">
+                </div>
+            </div>            
+        </section>
+
+        <section class="s_past">
+            <div class="wrapper_logo">
+                <h2>See the past DEFCON Hacker Conferences</h2>
+                <hr class="line">   
+                <p>Take a look at the past DEFCON hacker conferences which took place in Texas.</p>
+                <div>
+                    <img src="static/images/2021.jpg" alt="2011 conference image">
+                    <img src="static/images/defcon 2022.webp" alt="2013 conference image">
+                </div>
```

Comment:
- [ ] Kindly add text over the pictures to be similar to the original design 👇🏽 

| Original design | Your design |
|---|---|
| ![image](https://github.com/wacoo/DEF-CON_Summit/assets/58642949/038395c6-20ea-490f-9585-df93a1bca41a) | ![image](https://github.com/wacoo/DEF-CON_Summit/assets/58642949/df196969-6145-448b-9c4b-d3c73a12eee3) |